doc/user/layouts/partials/specifying-aws-credentials.html

```diff
@@ -9,10 +9,12 @@
    [`AWS_WEB_IDENTITY_TOKEN_FILE`][web-identity], and [many more][env-vars].
 3. `credential_process` command in the AWS config file, by default located at `~/.aws/config`.
 4. AWS credentials file and profile files. By default located at `~/.aws/config` and
-   `~/.aws/credentials` , respectively.
+   `~/.aws/credentials`, respectively.
 5. The [IAM instance profile][instance-profile] provided by the Instance Metadata Service. This
    will only work if running on an EC2 instance with an instance profile.
 
+*Changed in v0.10.0:* Materialize.
```

Comment:
Is this...intended?

Reply:
Ugh! https://github.com/MaterializeInc/materialize/pull/10658